jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java

```diff
@@ -21,40 +27,40 @@
 
     public static Label jdiTitle;
     // @UI("[ui=jdi-title]") public static Label jdiTitle;
-	public static Text jdiText;
+	public static com.epam.jdi.mobile.ui.html.elements.common.Text jdiText;
 	// @UI("[ui=jdi-text]") public static Text jdiText;
 
-    @UI("[value*='Red Button']") public static Button redButton;
+    @UI("[value*='Red Button']") public static com.epam.jdi.mobile.ui.html.elements.common.Button redButton;
     // @UI("input[type=button].red") public static Button redButton;
-    @UI("[value*='Disabled Button']") public static Button disabledButtonInput;
+    @UI("[value*='Disabled Button']") public static com.epam.jdi.mobile.ui.html.elements.common.Button disabledButtonInput;
     // @UI("input[type=button][disabled]") public static Button disabledButtonInput;
-    public static Button blueButton, dblClickButton, rightClickButton;
+    public static com.epam.jdi.mobile.ui.html.elements.common.Button blueButton, dblClickButton, rightClickButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4973196)